tests/LSP16UniversalFactory/LSP16UniversalFactory.test.ts

```diff
@@ -981,7 +984,7 @@ describe("UniversalFactory contract", () => {
         expect(caller).to.equal(context.universalFactory.address);
       });
 
-      it("should revert when deploying a proxy and sending value to a non payable function in deployCreate2Proxy", async () => {
+      it("should revert when deploying a proxy and sending value to a non payable function in deployERC1167Proxy", async () => {
```

Comment:
> sending value to a non payable function

The non-payable function is the function for initializing? If that's the case, maybe add 

```suggestion
      it("should revert when deploying a proxy and sending value to a non payable initializer function in deployERC1167Proxy", async () => {
```

(Although the function can be called anything, it might just be to give a bit more context)